sample/SampleServer/Program.cs

```diff
@@ -38,8 +38,8 @@ static async Task MainAsync(string[] args)
                     .WithOutput(Console.OpenStandardOutput())
                     .ConfigureLogging(x => x
                         .AddSerilog()
-                        .AddLanguageServer(LogLevel.Error)
-                        .SetMinimumLevel(LogLevel.Error))
+                        .AddLanguageServer()
+                        .SetMinimumLevel(LogLevel.Information))
```

Comment:
Maybe worth changing this to like `Debug` or `Trace` since `Information` is the default? Or is something else the default?

Reply:
You are correct, `Information` is the default level.